src/apple/device/simctl/mod.rs

```diff
@@ -27,16 +27,24 @@ impl Display for Device {
 
 impl<'a> From<Device> for AppleDevice<'a> {
     fn from(device: Device) -> AppleDevice<'a> {
+        let name = device.name.clone();
+        let is_visionos = name.contains("Apple Vision Pro");
+        let target = Target::for_triple(if cfg!(target_arch = "aarch64") {
+            match is_visionos {
+                true => "aarch64-apple-visionos",
+                _ => "aarch64-apple-ios", // TODO: figure out how to check for sim here, or probably do this elsewhere, and just add -sim to the triple
```

Comment:
this is the simctl module, if we're using this then it's always a simulator

Reply:
yeah, need to update this, we've done more stuff in our internal pipeline, and it's time to update it for 2.0.0 🥳